io/disk/ssd/blkdiscard.py

```diff
@@ -48,7 +48,8 @@ def setUp(self):
         smm = SoftwareManager()
         if not smm.check_installed("util-linux"):
             self.cancel("blkdiscard is needed for the test to be run")
-        self.disk = self.params.get('disk', default='/dev/nvme0n1')
+        device = self.params.get('disk', default='/dev/nvme0n1')
```

Comment:
@abdhaleegit Please make the `default=None`

Reply:
done taken care